features/users/user_create.feature

```diff
@@ -77,8 +77,10 @@ Feature: Sign Up for a new account
 
   Scenario: The user should be able to create a new account with a valid email and password
     When I fill in the sign up form with valid data
+    Then I should see the page title "Create Account | Example Archive"
       And all emails have been delivered
```

Comment:
Since this `And` step was under a `When` until we added the new `Then` step, it should get changed to a `When`.